jbmc/src/java_bytecode/java_bytecode_language.cpp

```diff
@@ -942,6 +944,8 @@ bool java_bytecode_languaget::typecheck(
       convert_single_method(
         method_sig.first, journalling_symbol_table, class_to_declared_symbols);
     }
+    convert_single_method(
+      INITIALIZE_FUNCTION, journalling_symbol_table, class_to_declared_symbols);
```

Comment:
What about old ci-lazy-methods?

Reply:
That is already indirectly tested via the regression tests. I chose to add a unit test of the new functionality in this PR, because it would otherwise be untested within this repository.

Reply:
I guess I could add a unit test of that...

Reply:
Added.